Formula/ocrmypdf.rb

```diff
@@ -0,0 +1,80 @@
+class Ocrmypdf < Formula
+  include Language::Python::Virtualenv
+
+  desc "Adds an OCR text layer to scanned PDF files"
+  homepage "https://github.com/jbarlow83/OCRmyPDF"
+  url "https://files.pythonhosted.org/packages/9b/c0/13a0ffb2184b85018b64466d1303ec1ceee05acff0b8ada16fdbf2ec101d/ocrmypdf-5.4.4.tar.gz"
+  sha256 "e523591c5d4e4a8cdeee2e9e24c3f631c8638a803dba74c5b8b274681784dbf4"
+
+  depends_on :python3
+  depends_on :x11 # Pillow needs XQuartz
+  depends_on "pkg-config" => :build
+  depends_on "tesseract"
+  depends_on "ghostscript"
+  depends_on "unpaper"
+  depends_on "qpdf"
+
+  # For Pillow source install
```

Comment:
You'll want to do something like [this](https://github.com/Homebrew/homebrew-core/blob/c9df884912456d09ebc19eb9cd7d4cc49b63a333/Formula/weboob.rb#L128-L143) for `pillow`, even though it's hideous.

Reply:
It's quite possible the various difficulties I had with pillow (like the failures someone reported that was fixed by adding `depends_on zlib`) are due to not having that black magic in place. I'll revise accordingly.